util/argo/argo_test.go

```diff
@@ -456,6 +456,115 @@ func TestValidateRepo(t *testing.T) {
 	assert.Equal(t, kustomizeOptions, receivedRequest.KustomizeOptions)
 }
 
+// TestValidateRepoWithSourceHydrator tests that ValidateRepo uses the dry source for manifest generation
+// when source hydrator is configured. This tests lines 485-486 in argo.go.
```

Comment:
```suggestion
// when source hydrator is configured.
```

That comment will get out of date fast. :-)